app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1533,6 +1535,21 @@ h1 {
   }
 }
 
+.main-title-caption {
+  zoom: 1;
```

Comment:
Properties should be ordered bottom, font-size, left, line-height, opacity, padding, position, width, zoom